### ISSUE TITLE
NTR: HIV (AIDS) antibody test positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8705]
New term request HIV (AIDS) antibody test positive (segal_050804153945):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000345
- label: Anti-human immunodeficiency virus antibody positivity
- parents: [Positive bloodstream virus-specific antibody positivity (HP:0430059)](https://hpo.jax.org/app/browse/term/HP:0430059)
- pmid: 
- def: The presence of antibodies in the blood circulation that react against a component of the human immunodeficiency virus (HIV).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8705
- synonyms: Anti-HIV antibody positivity(NONE)
